tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -159,6 +159,25 @@ Docker run and auto remove
     ${output}=  Split To Lines  ${output}
     Length Should Be  ${output}  ${count}
 
+Docker run and auto remove with anonymous volumes and named volumes
+       ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+       Should Be Equal As Integers  ${rc}  0
+       ${output}=  Split To Lines  ${output}
+       ${count}=  Get Length  ${output}
+       ${suffix}=  Evaluate  '%{DRONE_BUILD_NUMBER}-' + str(random.randint(1000,9999))  modules=random
+       Set Test Variable  ${namedImageVol}  non-anonymous-image-volume-${suffix}
+       Should Be Equal As Integers  ${rc}  0
+       Set Test Variable  ${imageVolumeContainer}  I-Have-Two-Anonymous-Volumes-${suffix}
+       ${rc}  ${c5}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --rm --name ${imageVolumeContainer} -v ${namedImageVol}:/data/db -v /I/AM/ANONYMOOOOSE mongo bash
```

Comment:
could do with a check of the `mongo` image to ensure the `/data/db` path remains a defined volume. Unlikely to check and not a blocker for this PR.

Reply:
@hickeng we should actually be covered by this case in my `1-04-Docker-Create` test modification. That test is designed to ensure that the metadata after creation indicates that a image volume was present and that we are binding a named volume over it. It checks that a `volume` path is present for `/data/db` which indicates that it was either requested anonymously or is a image volume. and it checks for the `HostConfig.Binds` entry which indicates that we mounted a named volume for that path`

Reply:
Thinking about this further we will actually see my test fail whenever we achieve more metadata correctness. Forcing us to improve the test to also check that the specific path is there rather than the named volume name. That is a potential flaw in this test, but it may be important since it will force us to understand what has changed with the increased metadata correctness.